oauth2_http/java/com/google/auth/oauth2/StsTokenExchangeResponse.java

```diff
@@ -40,25 +40,33 @@
 import javax.annotation.Nullable;
 
 /**
- * Defines an OAuth 2.0 token exchange successful response. Based on
- * https://tools.ietf.org/html/rfc8693#section-2.2.1.
+ * Represents an OAuth 2.0 token exchange request to the Google Security Token Service (STS), as
```

Comment:
This should be updated -- looks like its the Request docs

Reply:
Ah yes! 🤦  Thanks for catching this!